rotkehlchen/tests/api/test_trades.py

```diff
@@ -97,6 +97,16 @@ def assert_okay(response):
         )
     assert_okay(response)
 
+    # Now filter by location with ignored_names
+    response = requests.get(
+        api_url_for(
+            rotkehlchen_api_server_with_exchanges,
+            'tradesresource',
+        ),
+        json={'location': 'binance', 'ignored_names': ['binance']},
+    )
```

Comment:
don't provide the extra argument. You don't need to change this queries

Reply:
Also with this test you are not testing the functionality. You need to create a test with two instances of an exchange. Then mock the method for the trades and check that it was called for one but not the other. When you mock a method with pytest you can see how many times it got called. We do this in the tests but if you have trouble doing let me know